README.md

```diff
@@ -447,6 +447,7 @@
 * [**Rust**: _Building a DNS server in Rust_](https://github.com/EmilHernvall/dnsguide/blob/master/README.md)
 * [**Rust**: _Writing Scalable Chat Service from Scratch_](https://nbaksalyar.github.io/2015/07/10/writing-chat-in-rust.html)
 * [**TypeScript**: _Tiny Package Manager: Learns how npm or Yarn works_](https://github.com/g-plane/tiny-package-manager)
+* [**Php**: Create your first composer packagist package: Learns how php composer works under the hood](https://blog.jgrossi.com/2013/creating-your-first-composer-packagist-package/)
```

Comment:
This tutorial is almost useless. There are other tutorials out there with better explanation on how to create the repository and what are the required parameters in composer.json file to make it a composer package. Take a look at the following one: https://medium.com/@jun.chenying/how-to-create-a-composer-packagist-package-bdd414b9efd6

it has all information you need and includes screenshots too

Reply:
Okay, I'll replace it with your link.